src/cli/lint_command.zig

```diff
@@ -22,15 +22,10 @@ const Options = @import("../cli/Options.zig");
 
 pub fn lint(alloc: Allocator, options: Options) !u8 {
     const stdout = std.io.getStdOut().writer();
-    // NOTE: everything config related is stored in the same arena. This
```

Comment:
Can you re-add this note in `ManagedConfig`?

Reply:
should be back now